### PR TITLE
Added info to non-code-contributing guide

### PR DESCRIPTION
**Pull request by alejandrox1**
Signed-off-by: Jorge Alarcon Ochoa <alarcj137@gmail.com>

### CODE REVIEW

**Comment by alejandrox1** [comment]
@qedrakmar @jonasrosland ptal :smiley:

**Comment by alejandrox1** [comment]
ptal
/assign @qedrakmar @jonasrosland

File: contributors/guide/non-code-contributions.md [hunk 1]
```diff
@@ -18,14 +18,15 @@ Are you interested in any of the roles below? Come chat with us [on Slack](https
 These are roles that either span the project as a whole, or span several areas of the project. Most of the roles below can be considered "good-first-roles".
 
 - Community education
-  - Answering questions on Discuss, Slack, StackOverflow, etc
-  - K8s Office Hours
-  - Meet Our Contributors
+  - Answering questions on [Discuss](https://discuss.kubernetes.io/), [Slack](http://slack.k8s.io/), [StackOverflow](https://stackoverflow.com/questions/tagged/kubernetes), etc
+  - K8s Office Hours - [Slack](https://kubernetes.slack.com/messages/C6RFQ3T5H/), [Discuss](https://discuss.kubernetes.io/tags/office-hours), [YouTube](https://www.youtube.com/playlist?list=PL69nYSiGNLP3azFUvYJjGn45YbF6C-uIg)
```
Comment (jonasrosland) [inline contributors/guide/non-code-contributions.md]: Can you add in https://github.com/kubernetes/community/blob/master/events/office-hours.md here?

File: contributors/guide/non-code-contributions.md [hunk 2]
```diff
@@ -18,14 +18,15 @@ Are you interested in any of the roles below? Come chat with us [on Slack](https
...
+  - Meet Our Contributors - [Slack](https://kubernetes.slack.com/messages/C8WRR2BB9/), [YouTube](https://www.youtube.com/playlist?list=PL69nYSiGNLP3QpQrhZq_sLYo77BVKv09F)
```
Comment (jonasrosland) [inline contributors/guide/non-code-contributions.md]: Can you add in https://github.com/kubernetes/community/blob/master/mentoring/meet-our-contributors.md here?

File: contributors/guide/non-code-contributions.md [hunk 3]
```diff
@@ -18,14 +18,15 @@ Are you interested in any of the roles below? Come chat with us [on Slack](https
 These are roles that either span the project as a whole, or span several areas of the project. Most of the roles below can be considered "good-first-roles".
 
 - Community education
```
Comment (jonasrosland) [inline contributors/guide/non-code-contributions.md]: Can we add in an item regarding the Community Meetings here, and add an item like "Show demos of interesting tech related to K8s" or similar?
https://github.com/kubernetes/community/blob/master/events/community-meeting.md

File: contributors/guide/non-code-contributions.md [hunk 4]
```diff
@@ -18,14 +18,15 @@ Are you interested in any of the roles below? Come chat with us [on Slack](https
...
-  - Answering questions on Discuss, Slack, StackOverflow, etc
-  - K8s Office Hours
-  - Meet Our Contributors
+  - Answering questions on [Discuss](https://discuss.kubernetes.io/), [Slack](http://slack.k8s.io/), [StackOverflow](https://stackoverflow.com/questions/tagged/kubernetes), etc
+  - K8s Office Hours - [Slack](https://kubernetes.slack.com/messages/C6RFQ3T5H/), [Discuss](https://discuss.kubernetes.io/tags/office-hours), [YouTube](https://www.youtube.com/playlist?list=PL69nYSiGNLP3azFUvYJjGn45YbF6C-uIg)
+  - Meet Our Contributors - [Slack](https://kubernetes.slack.com/messages/C8WRR2BB9/), [YouTube](https://www.youtube.com/playlist?list=PL69nYSiGNLP3QpQrhZq_sLYo77BVKv09F)
   - Onboarding new contributors
     - Capturing the experiences of “Fresh Eyes” in the project
-  - Getting more people to SIG-Docs
+  - Getting more people to [SIG-Docs](https://github.com/kubernetes/community/tree/master/sig-docs)
```
Comment (parispittman) [inline contributors/guide/non-code-contributions.md]: https://kubernetes.io/docs/contribute/ if you want to add a direct link to their contrib guide

File: contributors/guide/non-code-contributions.md [hunk 5]
```diff
@@ -18,14 +18,15 @@ Are you interested in any of the roles below? Come chat with us [on Slack](https
...
 - Outward facing community work (might be more CNCF-oriented)
```
Comment (parispittman) [inline contributors/guide/non-code-contributions.md]: can we say that meetups are managed by the CNCF so there is a clear line here. we've had a few folks that took the last contribex survey think that it was our responsibility to help with their processes, build more to the program, etc. 

https://www.meetup.com/pro/cncf

Comment (parispittman) [reply contributors/guide/non-code-contributions.md]: https://github.com/cncf/meetups

**Comment by mrbobbytables** [comment]
This could be done in a separate PR (probably good first issue  / help wanted 😄for non-code), but should [apply the style guide](https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md) to the doc (thinking the links etc).

File: contributors/guide/non-code-contributions.md (hunk 1 above)
Comment (alejandrox1) [reply contributors/guide/non-code-contributions.md]: done

File: contributors/guide/non-code-contributions.md (hunk 2 above)
Comment (alejandrox1) [reply contributors/guide/non-code-contributions.md]: done

File: contributors/guide/non-code-contributions.md (hunk 3 above)
Comment (alejandrox1) [reply contributors/guide/non-code-contributions.md]: done

File: contributors/guide/non-code-contributions.md (hunk 4 above)
Comment (alejandrox1) [reply contributors/guide/non-code-contributions.md]: done

File: contributors/guide/non-code-contributions.md (hunk 5 above)
Comment (alejandrox1) [reply contributors/guide/non-code-contributions.md]: done

**Comment by alejandrox1** [comment]
@jonasrosland @parispittman thank you for your comments!
Please tell me if the changes look good.

**Comment by alejandrox1** [comment]
@mrbobbytables created #3599 :slightly_smiling_face:

**Comment by jonasrosland** [comment]
I'm happy with these changes, thank you @alejandrox1 !

/lgtm

**Pull request merged**